app/controllers/v1/users/timeline_posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class V1::Users::TimelinePostsController < ApplicationController
+  before_action :pundit_user
+  before_action do
+    @page = set_page.to_i
+  end
+
+  def index
```

Comment:
Another place where you might be able to make it smaller

Reply:
Ditto. Implemented the same solution as friends_controller and mutual_friends_controller